iridauploader/parsers/common.py

```diff
@@ -54,8 +54,8 @@ def find_directory_list(directory):
     """
 
     # Checks if we can access to the given directory, return empty and log a warning if we cannot.
-    if not os.access(directory, os.W_OK):
-        raise exceptions.DirectoryError("The directory is not writeable, "
+    if not os.access(directory, os.R_OK):
```

Comment:
`not os.access(directory, os.R_OK)` is called many times in different places. I think it would be better to have this in a method called isReadable in a common place which can be passed a directory to check

Reply:
Good call. I refactored all those calls in the `parsers` module to now be in `parsers/common.py` file.

Reply:
Awesome thanks.